pkg/operator/ceph/csi/csi.go

```diff
@@ -84,6 +84,14 @@ func SetParams(clientset kubernetes.Interface) error {
 		return errors.Wrap(err, "unable to parse value for 'ROOK_CSI_ENABLE_GRPC_METRICS'")
 	}
 
+	csiEnableCSIHostNetwork, err := k8sutil.GetOperatorSetting(clientset, controllerutil.OperatorSettingConfigMapName, "CSI_ENABLE_HOST_NETWORK", "false")
+	if err != nil {
+		return errors.Wrap(err, "unable to determine if CSI Host Network is enabled")
+	}
+	if CSIParam.EnableCSIHostNetwork, err = strconv.ParseBool(csiEnableCSIHostNetwork); err != nil {
+		return errors.Wrap(err, "unable to parse value for 'CSI_ENABLE_HOST_NETWORK'")
```

Comment:
```suggestion
		return errors.Wrapf(err, "failed to parse value for 'CSI_ENABLE_HOST_NETWORK'")
```

Reply:
@leseb we don't need Wrapf here, right?

Reply:
No we don't indeed, I guess I was thinking we could pass the var from  `CSI_ENABLE_HOST_NETWORK` but nevermind :)